### PR TITLE
Add story for FileNodeTree

### LINKED ISSUE #191: Add story for the FileTreeNode component
We should add a storybook story for the `FileTreeNode` component.

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #191

### CODE REVIEW

File: stories/setup/config.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
 
 // Include application styles.
 import '../../src/styles.scss';
 // Apply some custom styles to storybook.
 import './styles.scss';
 
+// Import all the "free solid FontAwesome" icons
+library.add(fas);
```
Comment (kumar303) [inline stories/setup/config.ts]: This has the potential to get out of sync with the real app so I'd rather see it moved to a shared function. I think all app configuration can be moved to a shared function. Storybook doesn't need all of this but it should be harmless to run the code:

```diff
diff --git a/src/index.tsx b/src/index.tsx
index caa7053..76faff4 100644
--- a/src/index.tsx
+++ b/src/index.tsx
@@ -11,22 +11,8 @@ import './styles.scss';
 import App from './components/App';
 import configureStore from './configureStore';
 
-if (process.env.NODE_ENV === 'production') {
-  // The second parameter prevents the log level to be persisted in a cookie or
-  // localStorage.
-  log.setLevel(log.levels.INFO, false);
-} else {
-  log.setLevel(log.levels.DEBUG, false);
-}
-
-if (process.env.REACT_APP_SENTRY_DSN) {
-  Raven.config(process.env.REACT_APP_SENTRY_DSN, {
-    logger: 'client',
-  }).install();
-}
-
-// Import all the "free solid FontAwesome" icons
-library.add(fas);
+configureApplication();
 
 const store = configureStore();
 
```

Comment (willdurand) [reply stories/setup/config.ts]: ah thanks, I wanted to file an issue for that too or, at least, leave a comment about the ever-growing code in `src/index.tsx`, which isn't super testable...

Comment (willdurand) [reply stories/setup/config.ts]: I will probably try to do that first, likely in a separate patch.

**Comment by willdurand** [comment]
> Also, this could be done in a follow-up: I prefer to see stories all on one page when possible so one could use hot reloading to develop changes more easily. We were using [Checkfront/react-storybook-addon-chapters](https://github.com/Checkfront/react-storybook-addon-chapters) for this on the frontend but maybe there is a simpler way.

That makes sense. I did not see it in the current setup, so I thought we would keep things simple and use what Storybook recommended.

File: stories/setup/config.ts (hunk 1 above)
Comment (willdurand) [reply stories/setup/config.ts]: Patch here: https://github.com/mozilla/addons-code-manager/pull/228

**Comment by willdurand** [comment]
> **Edit:** I filed #226

I'll take care of it in a separate patch, unless someone disagrees.

**Comment by willdurand** [comment]
I am working on #226 right now, good timing. Thanks!

**Pull request merged**